src/ast/passes/codegen_llvm.cpp

```diff
@@ -2693,8 +2712,10 @@ void CodegenLLVM::createRet(Value *value)
     case ProbeType::kretfunc:
     case ProbeType::iter:
     case ProbeType::rawtracepoint:
-      b_.CreateRet(b_.getInt64(0));
-      break;
+      return 0;
+    default:
+      LOG(BUG) << "Unknown probetype";
+      return 0;
```

Comment:
Why add default? If we leave it out and cases are exhaustive, then we will get compiler warning if new probe types are not handled. Adding default silences the warning and turns a compile time warning into a run time failure

Reply:
Ha. I was avoiding another compiler warning about returning void in a non void function. I'll fix

Reply:
I basically moved the `LOG(BUG)` out of the switch so now we have both the compiler warning and the runtime failure.